amethyst_audio/src/output.rs

```diff
@@ -22,6 +22,14 @@ pub struct Output {
     pub(crate) device: Device,
 }
 
+impl Default for Output {
+    fn default() -> Self {
+        default_output_device()
+            .map(|re| Output { device: re })
+            .expect("No default output device")
```

Comment:
When does this fail? If it actually fails, I‘m not sure if it should have a Default impl.